src/framework/components/collision/component.js

```diff
@@ -1,7 +1,13 @@
+import { Quat } from '../../../core/math/quat.js';
+import { Vec3 } from '../../../core/math/vec3.js';
+
 import { Asset } from '../../asset/asset.js';
 
 import { Component } from '../component.js';
 
+const vec3 = new Vec3();
```

Comment:
typically we add _ to these, so when you read the code, you don't think it's a local variable.